components/event-service/Makefile

```diff
@@ -32,3 +32,7 @@ ci-release: build build-image push-image
 .PHONY: clean
 clean:
 	rm -f $(binary)
+
+.PHONY: path-to-referenced-charts
+path-to-referenced-charts:
+	@echo "resources/application-connector"
```

Comment:
Please add a newline in the end of the file.

Reply:
:+1: